hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/commit/files/PendingSet.java

```diff
@@ -47,8 +47,8 @@
  */
 @InterfaceAudience.Private
 @InterfaceStability.Unstable
-public class Pendingset extends PersistentCommitData {
```

Comment:
Ignoring the fact that case-changing classnames is something very unlikely to take reliably in macos unless you can delete the old one first, I'm unsure about this. Why. It's not really a "set" in terms of java lang collection. At the same time, I'm not that fussy about classnames, though I do want it to stay all lower in the filesystem.